x-pack/plugins/serverless_observability/public/components/side_navigation/index.tsx

```diff
@@ -98,7 +127,51 @@ const navigationTree: NavigationTreeDefinition = {
   footer: [
     {
       type: 'navGroup',
-      ...getPresets('management'),
+      id: 'projest_settings_project_nav',
```

Comment:
This nav group section needs to live inside the shared ux component as a new "preset".

I was going to work on this issue: https://github.com/elastic/kibana/issues/160942 which was adding this section.
If you want to continue and finish the task then we would need to move it there and import it in this file.

This would imply:

* Create a "nav_tree_preset" folder
* Move [this file](packages/shared-ux/chrome/navigation/src/ui/nav_tree_presets.ts) inside of it
* Create this nav group as a preset in a separate file and make it available as a new preset ("project_settings")
* Update storybook story
* Add some tests

Or if you prefer I will handle it in my following task

Reply:
@sebelga, I can finish it but once clarification

Just checked the mockup 

![Screenshot 2023-06-30 at 11 31 41](https://github.com/elastic/kibana/assets/3369346/f5ffbda8-6e19-4113-a032-0fe7f20a5599)

The preset doesn't include integrations and Fleet which are required for oblt. 

Are they missing for a reason in the preset? Should I include them

Reply:
Mmm that's interesting. We need to confirm if Observability is going to have a different Project settings than Search and other project. Then it does not make sense that SharedUX owns that part of the navigation.

What we do need is expose the cloud url through props ([here](https://github.com/elastic/kibana/blob/main/packages/core/chrome/core-chrome-browser/src/project_navigation.ts#L91)). We will have to read them from the `cloud_links` configs (exposed by `kibana.yml` config) plugin and inject them (e.g a new API exposed by `ChromeInternal` > Serverless > `setProjectCloudLinks()` similar to what we do today with other apis to the Chrome service).

I am doing something similar here: https://github.com/elastic/kibana/pull/160535

But if you prefer to leave this external links task out for now, I will add them later. We need to anticipate that those links will only be visible to users with certain roles.

cc @jasonrhodes @petrklapka

Reply:
> We need to confirm if Observability is going to have a different Project settings than Search and other project. 

In that case, can we unblock the work in this PR from being merged? Happy to contribute back if we need to exposed the project settings as a new preset. 

> But if you prefer to leave this external links task out for now, I will add them later. We need to anticipate that those links will only be visible to users with certain roles.

I believe it will be better to leave it out for now. I can create a ticket if there is none

Reply:
> In that case, can we unblock the work in this PR from being merged? Happy to contribute back if we need to exposed the project settings as a new preset.

Sure. Can we then remove the external links to cloud in this PR?

Reply:
removed here - https://github.com/elastic/kibana/pull/160866/commits/c7260b4488268153c03a3d8a77ce10ab0119c894 

The cloud links will be tackled as part of the [ticket](https://github.com/elastic/kibana/issues/160942) or should I create a separate one?

Reply:
Thanks. They will be part of resolving the issue. 👍